src/pages/User/UserStats.tsx

```diff
@@ -196,8 +196,8 @@ function UserStats() {
 
   return (
     <Grid container spacing={3}>
-      <Grid item xs={8}>
-        <Paper style={{ height: '500px' }}>
+      <Grid item xs={12} md={8}>
+        <Paper style={{ height: '510px' }}>
```

Comment:
Why did you decide to change the height here?

Reply:
Although far from perfect, this small change in the height made the graph look a lot better for mobile view.